example/jss/covid/toy/TruncatedPoisson.bl

```diff
@@ -11,12 +11,7 @@ model TruncatedPoisson {
     logf(realization, mean) {
       if (mean <= 0) return NEGATIVE_INFINITY
       if (realization < 3 || realization > 11) return NEGATIVE_INFINITY
-      val factor = new ArrayList(9) => [
- 		for (int i : 3..<11){
- 			add(exp(i * log(mean)))
- 		}
- 	]
-      val sumF=-log(factor.sum())
+      val double sumF = (3..11).map(int i | exp(i * log(mean))/factorial(i)).sum()
```

Comment:
I believe this is the right way to implement. By the way, is the Python interface developed? I am wondering whether I can make some contributions.